src/app/util/attribute-table.cpp

```diff
@@ -54,18 +54,19 @@ EmberAfStatus emberAfWriteAttributeExternal(EndpointId endpoint, ClusterId clust
     case EmberAfAttributeWritePermission::AllowWriteNormal:
     case EmberAfAttributeWritePermission::AllowWriteOfReadOnly:
         return emAfWriteAttribute(endpoint, cluster, attributeID, dataPtr, dataType,
-                                  (extWritePermission == EmberAfAttributeWritePermission::AllowWriteOfReadOnly), false);
+                                  (extWritePermission == EmberAfAttributeWritePermission::AllowWriteOfReadOnly), false, false);
     default:
         return (EmberAfStatus) extWritePermission;
     }
 }
 
 EmberAfStatus emberAfWriteAttribute(EndpointId endpoint, ClusterId cluster, AttributeId attributeID, uint8_t * dataPtr,
```

Comment:
Suggest using a new method `emberAfWriteAttributeSkipNonVolatile` to be much more explicit, which calls the version with isTemp.

This way, almost none of the accessors below need to be changed.